neasqc_wp61/models/quantum/circuit.py

```diff
@@ -156,7 +163,7 @@ def __init__(
         """
         super().__init__(
             n_qubits, n_layers, axis_embedding, observables,
-            output_probabilities, device_name, data_rescaling, **kwargs
+            device_name, output_probabilities, data_rescaling, **kwargs
         )
         self.parameters_shape = (n_layers, 4  * n_qubits)
 
```

Comment:
in this function, prefer the variable name target to tgt, more explicit is better and its not that many more letters.